.cirrus.yml

```diff
@@ -53,11 +53,12 @@ env:
     VM_IMAGE_NAME:           # One of the "Google-cloud VM Images" (above)
     CTR_FQIN:                # One of the "Container FQIN's" (above)
 
-    # Curl-command prefix for downloading task artifacts, simply add the
-    # the url-encoded task name, artifact name, and path as a suffix.
-    ARTCURL: >-
-        curl --fail --location -O
-        --url https://api.cirrus-ci.com/v1/artifact/build/${CIRRUS_BUILD_ID}
+    # Unless overwritten, download cache tarball from the build task.
+    CACHE_KEY: build-${DISTRO_NV}
```

Comment:
You probably should include `CIRRUS_BUILD_ID` in the key.

Reply:
Oh okay, thanks for the tip.  I would not have guessed that :sweat: